### DISCUSSION TITLE
nominate @tsctx to be a maintainer

### DISCUSSION THREAD

**Discussion by Uzlopak** [nodejs/undici#3108] [General]
@tsctx is in my opinion a very active contributor. 

He is providing for 5-6 months PRs to the undici project, which mostly get accepted:
https://github.com/nodejs/undici/pulls?q=author%3Atsctx+

He also provided atleast one PR to nodejs core to fix an issue undici
https://github.com/nodejs/undici/issues/2026#issuecomment-1999010939
https://github.com/nodejs/node/pull/51526

I thereby nominate @tsctx for the maintainer team :)

**Comment by mcollina** [comment]
invited

**Reply by tsctx** [reply]
Thank you for inviting me to join undici team :)
